curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a23a46c4dc89026019f8e4.md

```diff
@@ -0,0 +1,343 @@
+---
+id: 64a23a46c4dc89026019f8e4
+title: Step 35
+challengeType: 0
+dashedName: step-35
+---
+
+# --description--
+
+Sort the argument `arr` in ascending order using the `sort` array method and store it in a variable named `sortNumbers`. What this will do is that it will sort the dice roll results in ascending order.
```

Comment:
```suggestion
Sort the argument `arr` in ascending order using the `sort` array method and store it in a variable named `sortNumbers`. This will sort the dice roll results in ascending order.
```

Reply:
done